### PR TITLE
added css questions

### PR DESCRIPTION
**Pull request by Virajj28**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/css-quiz.ts [hunk 1]
```diff
@@ -1154,6 +1154,25 @@ const cssQuiz = [
       "The justify-content property is used to position the grid in the grid container along the x-axis.",
     Link: "https://www.freecodecamp.org/news/css-grid-tutorial-with-cheatsheet/#css-grid-architecture",
   },
+  {
+    Question: "In CSS, how to center a div",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1158]:
```suggestion
    Question: "In CSS, which of the following options is one way to center a div?",
```

File: src/data/css-quiz.ts [hunk 2]
```diff
@@ -1154,6 +1154,25 @@ const cssQuiz = [
...
+    Answer: "flexbox and margin:auto",
+    Distractor1: "flexbox and margin: 10px",
+    Distractor2: "flexbox and margin:50%",
+    Distractor3: "flexbox and margin:100%",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1162]: I think we need to be more explicit here

```suggestion
    Answer: "use  display: flex; on the parent container and  margin: auto; for the child container",
    Distractor1: "use  display: flex; on the parent container and  margin: 20px; for the child container",
    Distractor2: "use  display: flex; on the parent container and  margin: 50px; for the child container",
    Distractor3: "use  display: flex; on the parent container and  margin: 45%; for the child container",
```

File: src/data/css-quiz.ts [hunk 3]
```diff
@@ -1154,6 +1154,25 @@ const cssQuiz = [
...
+    Explanation:
+      "You can use flexbox and margin:auto to center a div in CSS.",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1164]:
```suggestion
      "To center a div with CSS, apply display: flex; on the parent container and  margin: auto; for the child container.",
```

File: src/data/css-quiz.ts [hunk 4]
```diff
@@ -1154,6 +1154,25 @@ const cssQuiz = [
...
+    Link: "https://www.freecodecamp.org/news/how-to-center-a-div-with-css-10-different-ways/"
+  },
+  {
+    Question: "In CSS, 3em units are equivalent to how much px unit?",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1168]:
```suggestion
    Question: "In CSS, 3em units are equivalent to how much in px units?",
```

File: src/data/css-quiz.ts [hunk 5]
```diff
@@ -1154,6 +1154,25 @@ const cssQuiz = [
...
+    Answer: "48",
+    Distractor1: "32",
+    Distractor2: "16",
+    Distractor3: "12",
+    Explanation: "Three em unit is equivalent to 40 pixels",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1173]:
```suggestion
    Explanation: "3em is equivalent to 48 pixels.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Virajj28** [comment]
@jdwilkin4  Please can you merge branch to main as committed changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**